bitbots_hcm/src/bitbots_hcm/fall_checker.py

```diff
@@ -120,28 +79,17 @@ def calc_fall_quantification(self, falling_threshold_orientation, falling_thresh
                 return abs(current_axis__gyro) * (1 - skalar)
         return 0
 
-    def check_fallen(self, smooth_accel, not_much_smoothed_gyro):
-        """Check if the robot has fallen and is lying on the floor. Returns animation to play, if necessary."""
-        # Checks if the robot is still moving.
-        if any(abs(n) >= 0.1 for n in not_much_smoothed_gyro):
-            return None
-
-        # Decides which side is facing downwards.
-        if smooth_accel[0] < -7:
-            rospy.loginfo("FALLEN TO THE FRONT")
-            return self.FRONT
-
-        if smooth_accel[0] > 7:
-            rospy.loginfo("FALLEN TO THE BACK")
-            return self.BACK
-
-        if smooth_accel[1] > 7:
-            rospy.loginfo("FALLEN TO THE RIGHT")
-            return self.RIGHT
+    def fit(self, x, y):
+        # we have to do nothing, as we are not actually fitting any model
+        pass
```

Comment:
Maybe we should print a log message in case someone tries to train this classifier.